### PR TITLE
apply test-runner

### PR DESCRIPTION
**Pull request by h000780**
## Checklist

- [ ] Read CONTRIBUTING.md([for v1](https://github.com/kintone-labs/kintone-ui-component/master/CONTRIBUTING.md), [for v0](https://github.com/kintone-labs/kintone-ui-component/v0_dev/CONTRIBUTING.md))
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed lint and test command.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,28 +31,26 @@
     "build:esm": "tsc -p tsconfig.esm.json && node ./scripts/webpack-esm.js",
     "build:storybook": "build-storybook -o ./docs/storybook/",
     "storybook": "start-storybook",
-    "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && karma start --coverage",
+    "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && web-test-runner",
```
Comment (TomokoMiyake) [inline package.json:34]: How will it be if we use --coverage option?
You set coverage: true on config file, so it won't be needed??

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
I could execute unit test.
- Please delete karma.config file.
- Are there any features on test-runner to set the cover target as same as setting on karma.config?

File: package.json (hunk 1 above)
Comment (h000780) [reply package.json:34]: Yes. 
The coverage in config is whether to analyze code coverage
https://modern-web.dev/docs/test-runner/cli-and-configuration/#cli-flags

Comment (TomokoMiyake) [reply package.json:34]: Got it!

File: web-test-runner.config.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (TomokoMiyake) [inline web-test-runner.config.js:1]: No need of this line.
```suggestion
```

File: web-test-runner.config.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+module.exports = config => {
+    return {
```
Comment (TomokoMiyake) [inline web-test-runner.config.js:3]: Delete unnecessary space.
```suggestion
    return {
```

File: web-test-runner.config.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+            threshold:{
+                statements: 90,
+                branches: 90,
+                functions: 90,
+                lines: 90
+            }
+        },
+    };
+}
+
```
Comment (TomokoMiyake) [inline web-test-runner.config.js:17]: Delete unnecessary line.
```suggestion
```

**Review by TomokoMiyake — CHANGES_REQUESTED** [review]
Please fix the review point.

**Review by TomokoMiyake — APPROVED** [review]
LGTM!
I understood that you changed the file name from .js to .mjs for using as ESmodule.
https://modern-web.dev/docs/test-runner/cli-and-configuration/#configuration-file

**Pull request merged**